src/ocamlorg/lib/config.ml

```diff
@@ -23,3 +23,8 @@ let topelevels_path =
   Sys.getenv_opt "OCAMLORG_TOPLEVELS_PATH"
   |> Option.map (fun x -> Result.get_ok (Fpath.of_string x))
   |> Option.value ~default:Fpath.(v "src" / "ocamlorg_toplevel" / "bin" / "js")
+
+let package_state_path =
+  Sys.getenv_opt "OCAMLORG_PKG_STATE_PATH"
```

Comment:
We should set this variable in the Dockerfile.

Reply:
Indeed, thanks